### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (span)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2317]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/)
#### Element path
.scriptmanagercard > .content > span
#### Snippet
`<span class="header">See all projects</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.31 (foreground color: #ffffff, background color: #9eadae, font size: 14.0pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

This is the ``See all projects`` button on the homescreen
